elections/committee-conflict-of-interest.md

```diff
@@ -0,0 +1,61 @@
+# Committee Conflicts of Interest
+
+**Effective date**: 2023-10-03 (following the announcement of the [2023 Steering Committee election][steering-2023-election-schedule] results)
+
+[steering-2023-election-schedule]: /elections/steering/2023/README.md#schedule
+
+At time of writing, the only known conflict of interest for serving on
+committees in the Kubernetes Community exists with the
+[Steering Committee](/committee-steering/README.md) and
+the [Code of Conduct Committee (CoCC)](/committee-code-of-conduct/README.md).
+
+## Steering Committee and Code of Conduct Committee
+
+**_Kubernetes Community members cannot serve on both committees simultaneously._**
+
+This conflict may arise in the following scenarios:
+
+- A standing Steering Committee member has accepted a nomination to serve on
+  the Code of Conduct Committee
+- A standing CoCC member has elected to run for a seat on the Steering
+  Committee
+- A Kubernetes Community member has been nominated for candidacy for both
+  committees
+
+There are several factors under consideration when formulating a reasonable
+process for resolving this conflict of interest:
+
+- The distinctness of the Steering and CoCC charters, not limited to how these
+  committees need to adjudicate on community matters
+- The Kubernetes Community's expectation of commitment in its committee members
+  for duration of their terms
+- Continuity of committees and the duties they discharge
+- Undue burden on existing committee members to balance workload through
+  committee membership transitions, not limited to onboarding/offboarding
+- Undue burden on Election Officers in supporting overlapping elections with
+  the potential to generate conflicts
+- Minimization of conflict of interest edge cases not otherwise described by
+  this document
+
+With these considerations in mind, the following rules exist to prevent or
+resolve membership conflicts of interest between Steering and the CoCC:
+
+<!-- TODO(committee-coi): Reflect this list in each committee's "Eligibility for candidacy" section -->
+
+- Loss of committee member that occurs due to this conflict of interest will
+  be resolved per existing election or committee vacancy rules
+- Candidates who are nominated for a seat on both Steering and CoCC within the
+  same 12-month period must:
+  - (if elected) accept a seat on the committee whose election concludes first
+  - be withdrawn from candidacy in second election to conclude
+- Standing Steering or CoCC members must have served on their committee for a
+  period of twelve (12) months to be considered eligible for candidacy in the
+  other committee's election
+- Kubernetes Community members who have recently voluntarily vacated a seat on either
+  Steering or CoCC without serving the complete term will be ineligible for candidacy for a period of three (3)
+  months
```

Comment:
So I'm going to challenge this a little bit, so I'm going to tag the folks who were suggesting something like this as I'd like their opinion (cc @katcosgrove @celestehorgan)

While I completely understand that we don't want folks jumping from committee to committee on a regular basis for reasons of continuity and burden on their fellow committee members, I fear that putting these kinds of hard restrictions on the election processes will have the possible unwanted side effects:
- Folks may choose not to run for a committee that they wish to serve on, because they might also hypothetically want to run in a different election within the 12 months if they are elected. This would have the effect of shrinking the possible candidate pools.
- These are unpaid, voluntary positions. Forcing someone to serve in a capacity that they no longer wish to is not only unfair to the individual, but it's unfair to the community as a whole who benefits from their service.
- Employer support for upstream work can change. While an employer may provide time to serve on one committee, they may not provide that time to serve on the other.. this again would hold someone in a position where they don't want to be.

I think in practicality, some of this could be reduced by having a greater staggering in the elections.. right now we have CoCC in August and Steering in September. If they were staggered more, then we wouldn't have any risk of concurrent election periods and less possibility of thrashing if folks move between committees. But having these long, forced ineligibility periods I feel puts strain on individuals who are serving the community in really important ways.

Very open to other suggestions that might better solve these issues.

Reply:
just curious, what is the spirit of this norm and why 3 months? it seems that it tries to avoid some specific thing to happen but I can´t understand what

Reply:
@cblecker:

Our intent with this phrasing was that no potential for conflict exists until a committee position is actually held, so a candidate would be able to run for two positions within a 12-month period assuming they were not successful in their first attempt. If that's unclear, I'm happy to be more explicit!

The 3-month timeout period after voluntarily leaving your seat is there to discourage people from doing that simply to attempt to campaign for a seat on a more professionally lucrative committee. Allowing for committee hopping is especially problematic for the CoCC, where losing one member early is significantly more impactful than it is for Steering due to the much smaller committee size, but it's unfair to Steering as well and to the community at large due to the instability it causes. However, if we think the risk involved in voluntarily stepping down to run in another election the candidate may lose is deterrent enough without the 3-month timeout, I won't fight too hard to keep it. The individual should still be required to have served 12 months of their existing term before being eligible to run for a seat on another committee without first stepping down, though.

Staggering the elections more would certainly help solve this issue, but how doable is that in actual practice?